src/Components/FilterForm/FilterForm.styles.js

```diff
@@ -14,9 +14,10 @@ export const FullWidthSelect = styled(Select)`
 `
 
 export const SearchArea = styled.div`
-  background: ${GREY_3};
+  background: #f2f2f2;
```

Comment:
We should be using colour values from `govuk-colours` where possible.

Reply:
Yes, this seems to be a conflict between what the ticket specifies and what's available from the gov-uk colours. This shade of grey isn't available in the greys provided by gov-uk, but is used in quick look and has been requested for this story :/

Reply:
OK, will arrange a conversation with Eddie to get the consistency right between the apps.  If we are set on having our own colours for components like this in future, we could add our own colour palette to new-nomis-shared (not suggesting we do this right now though).